Casks/a/agent-tars.rb

```diff
@@ -7,9 +7,22 @@
   desc "Multimodal AI agent for GUI interaction"
   homepage "https://github.com/bytedance/UI-TARS-desktop"
 
+  # recent releases include latest, pre-release
   livecheck do
     url :url
     regex(/^Agent[._-]TARS[._-]v?(\d+(?:\.\d+)+(?:[._-]alpha[._-]?\d+)?)$/i)
```

Comment:
```suggestion
    regex(/^Agent[._-]TARS[._-]v?(\d+(?:\.\d+)+(?:[._-](?:alpha|beta)[._-]?\d+)?)$/i)
```

While we're here, we should update the regex to also match beta versions [until a stable version appears].